gitops/ckcp/base/deployment.yaml

```diff
@@ -16,8 +16,19 @@ spec:
     spec:
       serviceAccountName: anyuid
       containers:
-        - image: ckcp
+        - image: ghcr.io/kcp-dev/kcp:976a437
```

Comment:
This is a commit of the main branch.
I would suggest that we stick to a release branch, currently 0.3 and aim to follow minor releases (every 4 weeks I believe). Obviously this does not prevent people to experiment with newer versions but it should provide confidence than things just work after git clone.
The latest git commit on the 0.3 branch is d2bec95

Reply:
Created this [PR](https://github.com/kcp-dev/kcp/pull/887) to get the images tagged with branch names... let see whether it gets accepted

Reply:
I personally prefer when dependencies are frozen and do not change unless an action happens on the repo. It avoids things breaking for unknown reasons, and having 2 developers starting from the same clone to have different behaviors.
Not a big deal though, as any decision we take on this matter can easily be changed in the future.

Reply:
I am not saying the opposite. I am saying two things:
- we should use release versions, i.e: d2bec95
- having the images with an additional label in ghcr helps with picking a suitable image without the need to check the git commits